### PR TITLE
Fix ordinals for norwegian languages

### PR DESCRIPTION
**Pull request by stianjensen**
## Pull Request Checklist

Thank you for taking the time to improve Arrow! Before submitting your pull request, please check all *appropriate* boxes:

- [x] 🧪  Added **tests** for changed code.
- [x] 🛠️  All tests **pass** when run locally (run `tox` or `make test` to find out!).
- [x] 🧹  All linting checks **pass** when run locally (run `tox -e lint` or `make lint` to find out!).
- [x] 📚  Updated **documentation** for changed code.
- [x] ⏩  Code is **up-to-date** with the `master` branch.

If you have *any* questions about your code changes or any of the points above, please submit your questions along with the pull request and we will try our best to help!

## Description of Changes
In Norwegian, there should be a period after the day of the month.

### CODE REVIEW

**Comment by krisfremen** [comment]
hi @stianjensen

thanks for contributing, we always appreciate any improvements to the locales.

can you add tests for the Norwegian languages locales? right now we have none!

cheers!

**Comment by stianjensen** [comment]
Added tests for both Norwegian locales now.

**Comment by systemcatch** [comment]
Hey @stianjensen we are getting the following tox errors in our CI runs.

```shell
=================================== FAILURES ===================================
  ______________________ TestNorwegianLocale.test_describe _______________________
  
  self = <tests.test_locales.TestNorwegianLocale object at 0x1120c8ef0>
  
      def test_describe(self):
  >       assert self.locale.describe("now", only_distance=True) == "instantly"
  E       AssertionError: assert 'nå nettopp' == 'instantly'
  E         - instantly
  E         + nå nettopp
  
  tests/test_locales.py:2560: AssertionError
  _____________________ TestNewNorwegianLocale.test_describe _____________________
  
  self = <tests.test_locales.TestNewNorwegianLocale object at 0x10fa08630>
  
      def test_describe(self):
  >       assert self.locale.describe("now", only_distance=True) == "instantly"
  E       AssertionError: assert 'no nettopp' == 'instantly'
  E         - instantly
  E         + no nettopp
  
  tests/test_locales.py:2616: AssertionError
```

File: tests/test_locales.py [hunk 1]
```diff
@@ -2552,3 +2552,115 @@ def test_plurals_mk(self):
...
+        assert result == "for en time siden"
+
+    def test_weekday(self):
+        dt = arrow.Arrow(2015, 4, 11, 17, 30, 00)
+        assert self.locale.day_name(dt.isoweekday()) == "lørdag"
+        assert self.locale.day_abbreviation(dt.isoweekday()) == "lø"
+
+
+@pytest.mark.usefixtures("lang_locale")
+class TestNewNorwegianLocale:
```
Comment (systemcatch) [inline tests/test_locales.py:2614]: This is duplicated.

File: tests/test_locales.py [hunk 2]
```diff
@@ -2552,3 +2552,115 @@ def test_plurals_mk(self):
         assert self.locale._format_timeframe("months", 11) == "11 ай"
         assert self.locale._format_timeframe("year", 1) == "бір жыл"
         assert self.locale._format_timeframe("years", 12) == "12 жыл"
+
+
+@pytest.mark.usefixtures("lang_locale")
+class TestNorwegianLocale:
+    def test_describe(self):
+        assert self.locale.describe("now", only_distance=True) == "instantly"
```
Comment (systemcatch) [inline tests/test_locales.py:2560]: I think this is what is causing the error.

**Review by systemcatch — CHANGES_REQUESTED** [review]

File: tests/test_locales.py (hunk 1 above)
Comment (stianjensen) [reply tests/test_locales.py:2614]: It's not a duplicate, we have two written languages in Norway which are quite similar, but both are official languages. So this class tests the locale 'nn-no', while the other tests 'nb-no'.

File: tests/test_locales.py (hunk 2 above)
Comment (stianjensen) [reply tests/test_locales.py]: Huh - how did I miss that 😅 

Updated!

File: tests/test_locales.py (hunk 1 above)
Comment (systemcatch) [reply tests/test_locales.py:2614]: Ahh ok I should have read the code closer, thanks for the explanation.

**Review by systemcatch — APPROVED** [review]

**Comment by systemcatch** [comment]
@stianjensen  let's see what everyone else says but the changes look fine to me.

**Review by krisfremen — APPROVED** [review]
LGTM.

Thank you @stianjensen!

:1st_place_medal: @systemcatch

**Review by anishnya — APPROVED** [review]
LGTM!

**Review by systemcatch — APPROVED** [review]

**Comment by systemcatch** [comment]
Hey @stianjensen can you either give us access to update your branch or bump it to be even with master yourself?

**Comment by stianjensen** [comment]
I'll rebase it.
